dbc/json/json_query.go

```diff
@@ -105,7 +105,7 @@ func (q *Query) Cursor(in toolkit.M) (dbox.ICursor, error) {
 
 		if !ok {
 			return nil, errorlib.Error(packageName,
-				modQuery, "Cursor", e.Error())
```

Comment:
the `e` is nil, accessing `e.Error()` would be impossible. that's what causing this error.